libs/core/src/lib/utils/decorators/apply-css-style.decorator.ts

```diff
@@ -0,0 +1,17 @@
+import { Hash, ELEMENT_REF_EXCEPTION } from '../public_api';
+
+export function applyCssStyle(target: any, propertyKey: string, descriptor: PropertyDescriptor) {
```

Comment:
I know that in general this function is descriptive, but could you describe with  `/** */` a little why we should use this function?

Reply:
done